winlogbeat/eventlog/wineventlog_experimental.go

```diff
@@ -256,7 +256,7 @@ func (l *winEventLogExp) openChannel(bookmark win.Bookmark) (win.EvtHandle, erro
 }
 
 func (l *winEventLogExp) Read() ([]Record, error) {
-	var records []Record
+	records := make([]Record, 0, l.maxRead)
```

Comment:
How is this modification related to the bugfix? Same question for L427 in 
winlogbeat/eventlog/wineventlog.go.

Reply:
This was to make `golangci-lint` to pass. 
This was the error: `golangci lint error: consider pre-allocating records`. In both places.

Reply:
Thanks Andrew for the details. I added the `//nolint` directive.